### PR TITLE
community[patch]: Update voyage.ts to pass input_type  (default null, or query or document)

### PR DESCRIPTION
**Pull request by nicolas-geysse**
Fixes # (issue)
Improve embedding quality

### CODE REVIEW

File: libs/langchain-community/src/embeddings/voyage.ts [hunk 1]
```diff
@@ -53,6 +58,7 @@ export class VoyageEmbeddings
 
   headers?: Record<string, string>;
 
+  inputType: string;
```
Comment (jacoblee93) [inline libs/langchain-community/src/embeddings/voyage.ts:61]: I think this needs a `?` to mark as optional?

**Comment by jacoblee93** [comment]
Thank you!

**Comment by nicolas-geysse** [comment]
You're very welcome !
I just realized that I may have forgotten to commit here
libs/langchain-community/src/embeddings/tests/voyage.int.test.ts
and here
docs/core_docs/docs/integrations/text_embedding/voyageai.mdx
I just commit for this new files, but as I'm a total noob with Github, I don't know where has been pushed these new modifications. (can you see them ???)

**Comment by jacoblee93** [comment]
> You're very welcome ! I just realized that I may have forgotten to commit here libs/langchain-community/src/embeddings/tests/voyage.int.test.ts and here docs/core_docs/docs/integrations/text_embedding/voyageai.mdx I just commit for this new files, but as I'm a total noob with Github, I don't know where has been pushed these new modifications. (can you see them ???)

Ah, no that doesn't seem to be on this PR. It seems reasonable but feel free to push if you'd like!

File: libs/langchain-community/src/embeddings/voyage.ts (hunk 1 above)
Comment (nicolas-geysse) [reply libs/langchain-community/src/embeddings/voyage.ts]: yes you absolutely right, I should have secured with   
inputType?: string; // Make inputType optional

In the VoyageEmbeddings Class

**Pull request closed without merging**